src/pages/home/ReportScreen.js

```diff
@@ -354,7 +355,13 @@ function ReportScreen({
         // It possible that we may not have the report object yet in Onyx yet e.g. we navigated to a URL for an accessible report that
         // is not stored locally yet. If report.reportID exists, then the report has been stored locally and nothing more needs to be done.
         // If it doesn't exist, then we fetch the report from the API.
-        if (report.reportID && report.reportID === getReportID(route) && !isLoadingInitialReportActions) {
```

Comment:
We shouldn't remove this condition. just add the new logic without breaking the current one

Reply:
sorry, I mixed up the `if` blocks